config/config.schema.json

```diff
@@ -854,6 +854,48 @@
           "required": [
             "tlsCertificatesIssuer"
           ]
+        },
+        "ocpVersions": {
+          "type": "object",
+          "properties": {
+            "defaultVersion": {
+              "type": "object",
+              "properties": {
+                "channelGroupName": {
+                  "type": "string"
+                },
+                "version": {
+                  "type": "string"
+                }
+              },
+              "required": ["channelGroupName", "version"],
+              "additionalProperties": false
+            },
+            "channelGroups": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "url": {
+                    "type": "string"
+                  },
+                  "channelGroupName": {
+                    "type": "string"
+                  },
+                  "minVersion": {
+                    "type": "string"
+                  },
+                  "maxVersion": {
+                    "type": "string"
+                  }
+                },
+                "required": ["url", "channelGroupName", "minVersion", "maxVersion"],
```

Comment:
maxVersion is optional. 
I.e, for personal dev envs we can configure CS without setting maxVersion so it can get the latest images.

Reply:
Sure thanks